src/fullnode.rs

```diff
@@ -224,6 +225,7 @@ impl Fullnode {
         ledger_path: &str,
         sigverify_disabled: bool,
         leader_rotation_interval: Option<u64>,
+        rpc_port: Option<u16>,
```

Comment:
We don't make other ports configurable. Why this one?  And if you're going to do this, can you add docs to describe the difference between `None` and `Some(0)`?  Or if it needs to be within FULLNODE_PORT_RANGE?

Reply:
On some nodes it would be natural to put the RPC port at 80, having this option is nice I feel

Reply:
The other ports, except Drone, get dynamically assigned. Rpc is static for client interaction/simplicity, but it makes sense for it to be configurable on your own node.
Fwiw, parity clients boot with json-rpc services on a default static port, but offer the ability to override with a custom assignment. I probably should have programmed ours that way in the first place.

There are TODOs to make the drone port configurable as well. I think this is a good idea (and it would help resolve a couple flaky tests that we're ignoring), but it has bigger implications than Rpc, since airdrops are currently a critical piece of node startup.

Reply:
I can certainly add some documentation!

Reply:
👍